src/main/kotlin/no/nav/familie/ef/sak/fagsak/FagsakRepository.kt

```diff
@@ -47,17 +46,20 @@ interface FagsakRepository : RepositoryInterface<FagsakDao, UUID>, InsertUpdateR
     fun finnMedEksternId(eksternId: Long): FagsakDao?
 
     // language=PostgreSQL
-    @Query("""SELECT fp.ident FROM fagsak_person_old fp
-                    WHERE fp.fagsak_id=:id
-                    ORDER BY fp.endret_tid DESC
-                    LIMIT 1""")
+    @Query("""SELECT pi.ident FROM fagsak f
+                JOIN person_ident pi ON pi.fagsak_person_id = f.fagsak_person_id
+              WHERE f.id=:id
+              ORDER BY pi.endret_tid DESC
+              LIMIT 1""")
     fun finnAktivIdent(id: UUID): String
 
     // language=PostgreSQL
     @Query("""
-        SELECT DISTINCT fagsak_id AS first, FIRST_VALUE(ident) OVER (PARTITION BY fagsak_id ORDER BY endret_tid DESC) AS second
-          FROM fagsak_person_old
-        WHERE fagsak_id IN (:ider)""")
+        SELECT DISTINCT f.id AS first, 
+            FIRST_VALUE(ident) OVER (PARTITION BY pi.fagsak_person_id ORDER BY pi.endret_tid DESC) AS second
```

Comment:
Jeg kan for lite sql til å skjønne om omskrivingen her er "riktig", men du har kontroll? :)

Reply:
Skal være riktig, men ska verifisere att den blir testet 👍

Reply:
✅